sources/OpenAL/OpenAL/Handles/DeviceHandle.gen.cs

```diff
@@ -0,0 +1,34 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// Generated using the OpenAL Soft headers and corresponding dependencies.
+// Original source is licensed under the LGPL 2.0 license. Please note that while bindings are able to be MIT due to
+// being header-derived only, implementations of the headers may have a less permissive license.
+using System;
+using System.Diagnostics;
+using System.Runtime.CompilerServices;
+using System.Runtime.InteropServices;
+
+namespace Silk.NET.OpenAL;
+
```

Comment:
Handles for AL objects as well?

Reply:
Looks like we haven't done this for OpenGL either, added this to the bindings tracking issue to do in a future PR.

Reply:
Turns out this is already on the tracking issue. Amazing!